### PR TITLE
Pass quality to code build process 

### LINKED ISSUE #9380: vscode about dialog don't show insiders label
After https://github.com/gitpod-io/gitpod/pull/8783/ now `ideAlias` is always `code` for insiders and stable.
We need to look for other ways to differentiate vscode insiders and stable

### PR DESCRIPTION
**Pull request by andreafalzetti**
## Description
Companion PR of https://github.com/gitpod-io/openvscode-server/pull/347

The editor name is misconfigured and not showing correctly when users are using the Insider version.

| Before | After |
|--------|-------|
|  Screenshot 2022-05-12 at 16 49 03      |   Screenshot 2022-05-12 at 16 49 13

Note: Ignore the light/dark mode

## Related Issue(s)

Fixes #9380

## How to test
https://gitpod.slack.com/archives/C01KGM9BH54/p1652369461003149?thread_ts=1652353405.265159&cid=C01KGM9BH54

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@andreafalzetti: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by jeanp413** [comment]
@andreafalzetti  just a few tips, I think we should rename `ideQuality` to `codeQuality` so that it aligned with `codeCommit`, and for updating `product.json` with `jq` you can do it locally in the openvscode-server repo to test it and then just copy the script over the dockerfile

**Comment by jeanp413** [comment]
Comment on the openvscode-server PR https://github.com/gitpod-io/openvscode-server/pull/347#discussion_r871842467

File: components/ide/code/leeway.Dockerfile
Comment (jeanp413) [inline components/ide/code/leeway.Dockerfile:60]: Here's an [example for applying changes in one line](https://github.com/VSCodium/vscodium/blob/e480df5b1f0039321c0953b621f7ef085bae9c91/prepare_vscode.sh#L77-L78) :smile:

File: components/ide/code/leeway.Dockerfile
Comment (jeanp413) [inline components/ide/code/leeway.Dockerfile:58]: nameShort and nameLong could be different

File: WORKSPACE.yaml
Comment (jeanp413) [inline WORKSPACE.yaml:11]: Just need to set this to `stable` and we can merge it!

Comment (andreafalzetti) [reply WORKSPACE.yaml:11]: Is there any order in which I should merge this PR vs https://github.com/gitpod-io/openvscode-server/pull/347?

Comment (jeanp413) [reply WORKSPACE.yaml]: Nope, I'll merge the openvscode-server PR now

Comment (andreafalzetti) [reply WORKSPACE.yaml]: value updated, waiting for approval and I can merge

**Review by jeanp413 — APPROVED** [review]

**Pull request merged**
